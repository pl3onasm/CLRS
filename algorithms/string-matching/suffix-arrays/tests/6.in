a
aibohphobia is an irrational fear or distrust of palindromes.