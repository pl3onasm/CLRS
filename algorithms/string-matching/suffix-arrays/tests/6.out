Pattern: a
Shifts (7): 10, 0, 26, 50, 15, 31, 21
Longest repeated substrings:
  2 times:  i
  2 times:  o
  2 times: al
  2 times: is
  2 times: r 
  2 times: st
Number of unique substrings: 1842
Longest palindromic substring: aibohphobia
